Maven Project Compilation Error

Maven project does not compile via terminal using `mvn install`. 

OS: macOS Mojave 10.14.2 
Maven Version: 3.6

`java -version` produces:

> java version "10.0.1" 2018-04-17
> Java(TM) SE Runtime Environment 18.3 (build 10.0.1+10)
> Java HotSpot(TM) 64-Bit Server VM 18.3 (build 10.0.1+10, mixed mode)

`mvn install` produces:
> tictactoe $ mvn install
> [INFO] Scanning for projects...
> [INFO] 
> [INFO] -------------------< cs.ualberta.cmput402:TicTacToe >-------------------
> [INFO] Building TicTacToe 1.0-SNAPSHOT
> [INFO] --------------------------------[ jar ]---------------------------------
> [INFO] 
> [INFO] --- maven-resources-plugin:2.6:resources (default-resources) @ TicTacToe ---
> [WARNING] Using platform encoding (UTF-8 actually) to copy filtered resources, i.e. build is platform dependent!
> [INFO] skip non existing resourceDirectory /Users/zachredfern/Documents/CMPUT/c402/assignments/a1/tictactoe/src/main/resources
> [INFO] 
> [INFO] --- maven-compiler-plugin:3.1:compile (default-compile) @ TicTacToe ---
> [INFO] Changes detected - recompiling the module!
> [WARNING] File encoding has not been set, using platform encoding UTF-8, i.e. build is platform dependent!
> [INFO] Compiling 3 source files to /Users/zachredfern/Documents/CMPUT/c402/assignments/a1/tictactoe/target/classes
> [INFO] -------------------------------------------------------------
> [ERROR] COMPILATION ERROR : 
> [INFO] -------------------------------------------------------------
> [ERROR] Source option 5 is no longer supported. Use 6 or later.
> [ERROR] Target option 1.5 is no longer supported. Use 1.6 or later.
> [INFO] 2 errors 
> [INFO] -------------------------------------------------------------
> [INFO] ------------------------------------------------------------------------
> [INFO] BUILD FAILURE
> [INFO] ------------------------------------------------------------------------
> [INFO] Total time:  1.080 s
> [INFO] Finished at: 2019-01-14T09:45:37-07:00
> [INFO] ------------------------------------------------------------------------
> [ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.1:compile (default-compile) on project TicTacToe: Compilation failure: Compilation failure: 
> [ERROR] Source option 5 is no longer supported. Use 6 or later.
> [ERROR] Target option 1.5 is no longer supported. Use 1.6 or later.

--- a/pom.xml
+++ b/pom.xml
@@ -18,5 +18,10 @@
         </dependency>
 
     </dependencies>
+
+    <properties>
+    	<maven.compiler.source>1.6</maven.compiler.source>
+    	<maven.compiler.target>1.6</maven.compiler.target>
+	</properties>
     
 </project>

